client/scripts/controllers/chain/near/account.ts

```diff
@@ -149,3 +86,55 @@ export class NearAccount extends Account<NearToken> {
     throw new Error('not valid on Near protocol');
   }
 }
+
+export class NearAccounts implements IAccountsModule<NearToken, NearAccount> {
+  private _Chain: NearChain;
+  private _store: AccountsStore<NearAccount> = new AccountsStore();
+  public get store() { return this._store; }
+  public readonly keyStore: BrowserLocalStorageKeyStore;
+
+  private _validators: INearValidators = {};
+  public get validators() { return this._validators; }
+
+  private _app: IApp;
+  public get app() { return this._app; }
+
+  constructor(app: IApp) {
+    this._app = app;
+    this.keyStore = new BrowserLocalStorageKeyStore(localStorage);
+  }
+
+  public get(address: string): NearAccount {
+    if (!this._Chain) return null; // We can't construct accounts if the NEAR chain isn't loaded
+    return this.fromAddress(address);
+  }
+
+  public fromAddress(address: string): NearAccount {
+    let acct;
+    try {
+      acct = this._store.getByAddress(address);
+    } catch (e) {
+      acct = new NearAccount(this.app, this._Chain, this, address);
+    }
+    return acct;
+  }
+
+  public async init(ChainInfo: NearChain): Promise<void> {
+    this._Chain = ChainInfo;
+    const validators = ChainInfo.nodeStatus.validators as unknown as INearValidator[];
```

Comment:
nodeStatus may not be available here. if so we should not be init'ing the chain, but we are doing so